kite-morphlines/kite-morphlines-core/pom.xml

```diff
@@ -62,6 +62,11 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+    	<groupId>commons-codec</groupId>
+    	<artifactId>commons-codec</artifactId>
```

Comment:
The policy for existing morphline mvn modules is to not add additional dependencies on further 3rd party libs, in order to avoid the potential for dependency version conflicts / jar hell. Consider providing similar functionality just using what's in the JDK or with existing dependencies, or consider adding a new separate mvn module, e.g. along the lines of the kite-morphlines-json module.